geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/DescribeMappingCommand.java

```diff
@@ -84,12 +85,23 @@ public ResultModel describeMapping(@CliOption(key = DESCRIBE_MAPPING__REGION_NAM
       if (groups == null) {
         groups = new String[] {ConfigurationPersistenceService.CLUSTER_CONFIG};
       }
+      ArrayList<String> groupsArray = new ArrayList<String>();
+      boolean isProxyRegion = false;
       for (String group : groups) {
         CacheConfig cacheConfig = getCacheConfig(configService, group);
         RegionConfig regionConfig = checkForRegion(regionName, cacheConfig, group);
+        if (regionConfig.getType().equals("PARTITION_PROXY")
```

Comment:
Have describe and create use a common method to determine if the clusterconfig region is a proxy